modules/search/class.jetpack-search.php

```diff
@@ -429,6 +429,55 @@ public function search( array $es_args ) {
 		return $response;
 	}
 
+	/**
+	 * Displays HTML about the result of the spelling auto-correction.
+	 * Should be called during loop_start to be displayed below page headers and above post results.
+	 *
+	 * @since 6.4.2
+	 *
+	 * @param string $spell_text The text to show in the suggest block
+	 */
+	public function display_suggest_result( $spell_text ) {
+		printf("<div class=\"suggest-result page-header\"><h4>" . $spell_text . "<br><br></h4></div>");
```

Comment:
The output of $spell_text should be getting sanitized. See: https://codex.wordpress.org/Validating_Sanitizing_and_Escaping_User_Data

I think the text that is output should probably also be here so that the localization can be done correctly: https://codex.wordpress.org/I18n_for_WordPress_Developers

Nitpick: spaces after/before parens. You got it lots of other places though. :)